claimManagement/src/main/java/org/openimis/imisclaims/EnquireActivity.java

```diff
@@ -152,7 +160,8 @@ public void onClick(View v) {
                 inputManager.hideSoftInputFromWindow(getCurrentFocus().getWindowToken(), InputMethodManager.HIDE_NOT_ALWAYS);
 
                 ClearForm();
-                if (!CheckCHFID()) return;
+                Escape escape = new Escape();
+                if (!escape.CheckCHFID()) return;
```

Comment:
call escape.CheckCHFID(etCHFID.getText()) OR escape.CheckCHFID(tvCHFID.getText()) (don't know which one is here)